src/iomusxml.cpp

```diff
@@ -3640,4 +3670,18 @@ void MusicXmlInput::ShapeFermata(Fermata *fermata, pugi::xml_node node)
     }
 }
 
+std::string MusicXmlInput::GetOrnamentGlyphNumber(const std::string &attributes) const
+{
+    static std::map<std::string, std::string> precomposedNames = {
```

Comment:
@lpugin I know you've mentioned that **static** should not be used. This can be changed either to non-static variable, but that would lead to memory allocation on each entry. Alternatively it could be changed to if-else statments, but it doesn't seem the best approach either.
Should I remove **static** regardless?